helm/defectdojo/templates/initializer-job.yaml

```diff
@@ -2,7 +2,7 @@
 apiVersion: batch/v1
 kind: Job
 metadata:
-  name: {{ .Release.Name }}-initializer
+  name: {{ .Release.Name }}-initializer-{{ randAlphaNum 5 | lower }}
```

Comment:
how about a timestamped extension (to the second)? would allow easy find and easy purge by date

Reply:
We can, even I don't see it so important.

Reply:
my understanding is that it'll create a new job for each release? it may get messy in the end. it's no big deal but would be nice to be able to easily identify those jobs
something like that could work: 

`now | date "2006-01-02_15.04.05"`
(http://masterminds.github.io/sprig/date.html / https://www.pauladamsmith.com/blog/2011/05/go_time.html )

Reply:
@alles-klar what's your thoughts on this? last point remaining (then rebase is required)

Reply:
@ptrovatelli sorry I was away this days, do you want me something?

Reply:
@dsever yes the last point that we discussed is about having jobs named per date to easily find them for logs and purge them. i have given an example of code using go (the language used by templating) although I havent tested it.

Reply:
rebased is required too

Reply:
@ptrovatelli, 

Init name is now in format: defectdojo-initializer-2020-08-11-23-31